### ISSUE TITLE
🐛 Screenshots section in issue templates

### ISSUE THREAD

**Issue by bradystroud** [SSWConsulting/SSW.GitHub.Template#6]
@adamcogan 
Hi @pierssinclairssw 

### Describe the Bug
Currently our issue templates have a screenshot section

### Expected Behavior
Screenshots should go wherever they are relevant in the issue.

### Tasks
- [ ] If you agree, remove the screenshot section from issue templates

### Screenshots
![image](https://user-images.githubusercontent.com/38869720/176120809-a5d94ae0-12c1-4fa7-9a64-4d9d90b135bd.png)
**Figure: This screenshot would have made more sense in the Describe the Bug section. It has no context here**

Thanks!

**Comment by jernejk** [comment]
I agree with @bradystroud. We want to avoid scrolling up and down, trying to find the matching screenshot, to understand the issue/feature.

Having a separate section for details is still very beneficial, with additional screenshots, links, references to other GH issues, videos and details that are nice to know for developers but not necessary for understanding the issue.
